pubsub_test.ts

```diff
@@ -39,10 +42,19 @@ test({
       message: "wayway",
     });
     await sub.close();
-    const a = await redis.get("aaa");
-    assertEquals(a, undefined);
-    pub.close();
-    redis.close();
+
+    assertEquals(sub.isClosed, true);
+    assertEquals(redis.isClosed, true);
+
```

Comment:
How about the following changes?
```suggestion
await pub.close();
await assertThrowsAsync(async () => {
  await redis.get("aaa");
}, Deno.errors.BadResource);
```